### ISSUE TITLE
System.Format Exception when opening the CPU view on on a Linux Trace

### ISSUE THREAD

**Issue by sandeeprawat** [microsoft/perfview#1048]
I'm getting the System.Format exception when trying to load .net core 2.1 trace from Linux. Using latest PerfView (2.0.46)

Shared trace at https://drive.google.com/open?id=1z-WOW_jjoOz_a7GKEjgHrbQFluSA0lfQ

```
Completed: Opening CPU Stacks (Elapsed Time: 115.629 sec)
Exception Occurred: System.AggregateException: One or more errors occurred. ---> System.FormatException: Input string was not in a correct format.
at System.Number.ParseDouble(String value, NumberStyles options, NumberFormatInfo numfmt)
at Diagnostics.Tracing.StackSources.LinuxPerfScriptEventParser.d__30.MoveNext()
at Diagnostics.Tracing.StackSources.LinuxPerfScriptEventParser.d__9.MoveNext()
at Diagnostics.Tracing.StackSources.ParallelLinuxPerfScriptStackSource.<>c__DisplayClass1_1.b__0(Object givenArrayIndex)
at System.Threading.Tasks.Task.Execute()
```

**Comment by bozila** [comment]
I have the same issue. .netcore 2.2.8 in kubernetes linux pod. perfview version is 2.0.55.

**Comment by jaredrsowers** [comment]
Same error on 2.0.58 from a 3.1.7 trace from a k8s pod.

> Started: Opening CPU Stacks
> Completed: Opening CPU Stacks   (Elapsed Time: 0.702 sec)
> Exception Occurred: System.FormatException: Input string was not in a correct format.
>    at System.Number.ParseDouble(String value, NumberStyles options, NumberFormatInfo numfmt)
>    at Microsoft.Diagnostics.Tracing.StackSources.LinuxPerfScriptEventParser.<NextEvent>d__30.MoveNext()
>    at Microsoft.Diagnostics.Tracing.StackSources.LinuxPerfScriptEventParser.<Parse>d__9.MoveNext()
>    at Microsoft.Diagnostics.Tracing.StackSources.LinuxPerfScriptStackSource.DoInterning()
>    at Microsoft.Diagnostics.Tracing.StackSources.LinuxPerfScriptStackSource.InternAllLinuxEvents(Stream stream)
>    at Microsoft.Diagnostics.Tracing.StackSources.LinuxPerfScriptStackSource..ctor(String path, Boolean doThreadTime)
>    at PerfView.LinuxPerfViewData.OpenStackSourceImpl(String streamName, TextWriter log, Double startRelativeMSec, Double endRelativeMSec, Predicate`1 predicate) in C:\A\_work\68\s\src\PerfView\PerfViewData.cs:line 8404
>    at PerfView.PerfViewStackSource.OpenStackSource(String streamName, TextWriter log, Double startRelativeMSec, Double endRelativeMSec, Predicate`1 predicate) in C:\A\_work\68\s\src\PerfView\PerfViewData.cs:line 3890
>    at PerfView.PerfViewStackSource.<>c__DisplayClass23_0.<Open>b__0() in C:\A\_work\68\s\src\PerfView\PerfViewData.cs:line 3914
>    at PerfView.StatusBar.<>c__DisplayClass22_0.<StartWork>b__0() in C:\A\_work\68\s\src\PerfView\GuiUtilities\StatusBar\StatusBar.xaml.cs:line 254
> An exceptional condition occurred, see log for details.

**Comment by brianrob** [comment]
This has been fixed in master, with #1244.  It will be included in the next release.
